### ISSUE TITLE
A11y_AzureDataStudio_New deployment: SQL Server 2019 Big Data Cluster_ Screen reader: Narrator is not announcing the role of Dropdown button for the Location field.

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/azuredatastudio#18593]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info & Relationships; #Narrator; #NVDA;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on the Narrator.
2 Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment"
5.  In the Deployment options, select "SQL Server Big Data Cluster" card.
6. Validate the required tools are installed and the "Required Tools" table shows the status as "Installed". Without the required tools the Select button will be disabled.
7. Select the checkbox to accept the license terms and click Select
8. In the Deployment profiles screen, select an appropriate profile and click Next (you can also leave the defaults on)
9. In the Azure settings page, enter the required details and click Next
10.Verify whether that the narrator is announcing the role of dropdown or not.

### Actual Result:
 Narrator is not announcing the role of Dropdown button for the Location field.

### Expected Result:
Narrator should announced the role of Dropdown button for the Location field.

### Attachment:
**Narrator Attachment:**
![A11y_AzureDataStudio_New deployment_ SQl server 2019 big data cluster_DropDown_Screen Reader](https://user-images.githubusercontent.com/99962579/156160814-79a05c1b-00de-4c6c-aff2-71f4654f11cc.gif)
**NVDA Attachment:**
![A11y_AzureDataStudio_New Deployment_SQL server 2019 big data cluster _Dropdown_Screen Reader](https://user-images.githubusercontent.com/99962579/156161470-c9167da5-1ad0-4443-b372-d4ce99962e31.gif)

**Comment by Charles-Gagnon** [comment]
As your gif shows, this is correctly being announced as "combo edit" so I'm not sure what the issue is here - this is indeed an editable combo box input so that should be correct.

**Comment by VijayaBharathyA** [comment]
@Charles-Gagnon 
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**
Could you please tell us how to get "SQL Server Big Data Cluster in Deployment options to verify this bug
![image](https://user-images.githubusercontent.com/100189407/218678617-122733b8-9292-4379-91d2-3093e24f314e.png)

**Comment by Charles-Gagnon** [comment]
The Big Data Cluster feature has been removed, so all issues related to that functionality should be closed.

**Comment by VijayaBharathyA** [comment]
#Closed;
Since Big Data Cluster feature has been removed we are closing bug
